### PR TITLE
fix: log worker errors for startProctoredExam

### PR DESCRIPTION
**Pull request by bseverino**
[MST-1162](https://openedx.atlassian.net/browse/MST-1162)

### CODE REVIEW

File: src/data/thunks.js [hunk 1]
```diff
@@ -150,10 +150,22 @@ export function startProctoredExam() {
         .then(() => updateAttemptAfter(
           exam.course_id, exam.content_id, continueAttempt(attempt.attempt_id),
         )(dispatch))
-        .catch(() => handleAPIError(
-          { message: 'Something has gone wrong starting your exam. Please double-check that the application is running.' },
-          dispatch,
-        ));
+        .catch(error => {
+          if (error) {
+            logInfo(
```
Comment (schenedx) [inline src/data/thunks.js:155]: Why logInfo instead of just logError here?

**Review by schenedx — COMMENTED** [review]
Got a question. Nice test setup and checking!

File: src/data/thunks.js (hunk 1 above)
Comment (bseverino) [reply src/data/thunks.js:155]: I thought about it initially, but I'm afraid it'll blow up the error rate, especially if there is user error involved.

**Review by Zacharis278 — APPROVED** [review]
not totally familiar with how our frontend logging works, can you brief me how we can find these?

**Comment by bseverino** [comment]
@Zacharis278 Yeah, there's a pretty good overview here: https://github.com/edx/frontend-platform/blob/4c9057b5796a8cf9090eb0f020904c2ef547c1cc/src/logging/NewRelicLoggingService.js#L43

File: src/data/thunks.js (hunk 1 above)
Comment (schenedx) [reply src/data/thunks.js:155]: It's a fair concern. I still think we should do logError. I thought we can just exclude this one on newrelic? Or at least set the alert threshold high?
To me, this is a good error to alert us anyways to actively help and identify learner issues

**Review by schenedx — APPROVED** [review]
Either way, I am good with this change.

**Pull request merged**